packages/react/src/SelectPanel/SelectPanel.module.css

```diff
@@ -31,3 +36,45 @@
   height: inherit;
   max-height: inherit;
 }
+
+.Message {
+  display: flex;
+  height: 100%;
+  min-height: min(calc(var(--max-height) - 150px), 324px); /* maxHeight of dialog - (header & footer) */
+  padding-top: var(--base-size-24);
+  padding-bottom: var(--base-size-24);
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  gap: var(--base-size-4);
+
+  a {
+    color: inherit;
+    text-decoration: underline;
+  }
+}
+
+.MessageTitle {
+  font-size: var(--text-body-size-medium);
+  font-weight: var(--base-text-weight-semibold);
+}
+
+.MessageBody {
+  display: flex;
```

Comment:
do we need the display: flex styles here? The MessageBody should read + wrap as a sentence. 

Current:
<img width="378" alt="Screenshot 2025-03-10 at 1 35 50 PM" src="https://github.com/user-attachments/assets/ce3bc0ad-9cdd-47cb-95db-3ac755224b8b" />


Expected:
<img width="365" alt="Screenshot 2025-03-10 at 1 36 33 PM" src="https://github.com/user-attachments/assets/a4fbaf9f-208a-4944-adb9-d4fa1a2da5e1" />

Reply:
done in [5b2f4fe](https://github.com/primer/react/pull/5142/commits/5b2f4fe82faccd84f29f99249aa455d2144fed20)!